scanpy_scripts/cmds.py

```diff
@@ -210,3 +214,31 @@
     arg_desc=_IP_DESC,
     opt_set='plot_paga'
 )
+
+COMBAT_CMD = make_subcmd(
+    'combat',
+    combat,
+    cmd_desc='ComBat function for batch effect correction',
+    arg_desc=_IO_DESC
+)
+
+HARMONY_INTEGRATE_CMD = make_subcmd(
+    'harmony_integrate',
```

Comment:
I feel "harmony_integrate" is a bit long. I might have missed something but is "harmony" ok?

Reply:
There's also 'harmony-timeseries' in Scanpy, so I thought I'd use the full function name to be clear. But I guess being a subfunction of integrate should be clear enough